Source/RenderStreamEditor/Private/RenderStreamEditorModule.cpp

```diff
@@ -830,7 +835,9 @@ void FRenderStreamEditorModule::OnObjectPostEditChange(UObject* Object, FPropert
     {
         // we only care if default objects have been changed eg. project settings objects like UGameMapSettings
         // add include/exclude filters here if required
-        DirtyAssetMetadata = true;
+        if (Object->IsA<UGameMapsSettings>() || Object->IsA<URenderStreamSettings>()) {
+            DirtyAssetMetadata = true;
+        }
```

Comment:
This is to prevent it from regenerating the schema every time any project setting is changed in unreal and limiting it to only relevant ones